WordPress/Classes/ViewRelated/Blog/SharingConnectionsViewController.m

```diff
@@ -196,7 +196,7 @@ - (void)configurePublicizeCell:(UITableViewCell *)cell atIndexPath:(NSIndexPath
     PublicizeConnection *connection = [[self connectionsForService] objectAtIndex:indexPath.row];
     cell.textLabel.text = connection.externalDisplay;
 
-    if ([connection isBroken]) {
+    if ([connection isBroken] || [connection mustDisconnect]) {
```

Comment:
I tend to encapsulate these boolean checks into a single method. In my experience, that makes the code more readable, in the long term, because it provides a name that can clearly explain what is happening and why it is important. 

In this case, also, I think it might make sense to encapsulate that logic in the `PublicizeConnection` object as well, so this view controller does not need to know about `isBroken` and / or `mustDisconnect`.

What do you think?